ui/src/wizard/AzureWizard.ts

```diff
@@ -91,7 +88,7 @@ export class AzureWizard<T> implements types.AzureWizard<T>, IInternalAzureWizar
         await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification }, async progress => {
             let currentStep: number = 1;
 
-            const steps: AzureWizardExecuteStep<T>[] = getExecuteSteps(this._wizardNode);
+            const steps: AzureWizardExecuteStep<T>[] = this._executeSteps.sort((a, b) => b.priority - a.priority);
```

Comment:
Should you warn about dup priorities?

Reply:
Not sure what you mean by "warn":
- I don't want to do `vscode.window.showWarningMessage` because I'm guessing most cases will work just fine even if there's duplicate priorities and this feels like it would "block" those cases
- I'm not opposed to `console.warn`, but I wouldn't add it myself. Just a personal preference to avoid `console.*` in production code